src/test/java/com/flightstats/hub/dao/aws/S3ConfigStrategyTest.java

```diff
@@ -39,7 +39,8 @@ private void verify(int channels, int iterations) {
         for (int i = 0; i < channels; i++) {
             String name = StringUtils.randomAlphaNumeric(10);
             channelConfigs.add(ChannelConfig.builder().name(name).build());
-            names.add(name);
+            String batchedName = name + "Batch";
```

Comment:
Do we have a centralized location of this magic-ish string format? I'm guessing we use it in more than one place...

Reply:
We didn't really, so I added it